mmv1/products/networkservices/EdgeCacheOrigin.yaml

```diff
@@ -291,7 +291,7 @@ properties:
 
               You may add a maximum of 5 request headers.
```

Comment:
This part of the description would contradict the new maximum, so I think we should update it to 25 here too, or perhaps remove this part of the description so that we don't have to keep updating it in the future.

Reply:
Good catch. let me update the descriptions then. Thanks @roaks3